building/glass/debclean.py

```diff
@@ -0,0 +1,49 @@
+import os
+import shutil
+
+
+def clean_paths(rootfs, *paths):
```

Comment:
This module seems like a good way to accidentally wipe a developer's filesystem.  I wonder if there's a good way to check so that it fails if not called inside the chroot?

Reply:
I misread the surrounding code.  Nonetheless this would still be bad if called with the wrong `rootfs` argument, no?

Reply:
Yes... but so would a lot of things? Like shutil.rmtree, for example.

We have two layers of protection: 1. the build chroot and 2. the fact that this is embedded in a larger program that can ensure it's used correctly.

Both of these would have to fail for it to cause serious damage to a developer's system... and that's why we have backups.